manager.go

```diff
@@ -89,38 +87,20 @@ func (m *Manager) LoadConfig(cfgPath string) error {
 		}
 	}
 
-	m.cloudAddr = cloud["app_address"]
-	m.partID = cloud["id"]
-	m.cloudSecret = cloud["secret"]
+	m.cloudConfig = &logging.CloudConfig{
+		AppAddress: cloud["app_address"],
+		ID:         cloud["id"],
+		Secret:     cloud["secret"],
+	}
 
 	return nil
 }
 
-// create and attach a NetAppender.
-func (m *Manager) attachNetAppender() error {
-	if m.conn == nil {
-		return errors.New("NetAppender requires non-null conn")
-	}
-	if m.netAppender != nil {
-		return errors.New("Manager already has non-nil netAppender")
-	}
-	if m.partID == "" {
-		return errors.New("can't create NetAppender without part ID")
-	}
-	netAppender, err := logging.NewNetAppender(
-		&logging.CloudConfig{
-			AppAddress: m.cloudAddr,
-			ID:         m.partID,
-			Secret:     m.cloudSecret,
-		},
-		m.conn,
-	)
-	if err != nil {
-		return err
+func (m *Manager) CreateNetAppender() (*logging.NetAppender, error) {
+	if m.cloudConfig == nil {
+		return nil, errors.New("can't create NetAppender before config has been loaded")
 	}
-	m.netAppender = netAppender
```

Comment:
Don't you still need this line? Without it, I can't find how/where m.NetAppender is ever actually set.